lms/views/basic_lti_launch.py

```diff
@@ -70,7 +69,7 @@ def sync_lti_data_to_h(self):
         """
 
         self.request.find_service(name="lti_h").sync(
-            [self.context.h_group], self.request.params
```

Comment:
We should change the signature and docs here (they were wrong before this PR):

https://github.com/hypothesis/lms/blob/refactor-h-group/lms/services/lti_h.py#L28

Probably `groupings` instead of h_groups?

Reply:
I think this is true, but it's also quite a big change on it's own (as it passes all the way through that class. I think I'll do it in a separate PR.